### PR TITLE
updates per new public key options for IdPs

### PR DESCRIPTION
**Pull request by susanharper-okta**
## Description:
- **What's changed?** added step for selecting public key/private key
- **Is this PR related to a Monolith release?** Yes 2022.06.0

NETLIFY link: https://628ff9090444810e3d67b083--reverent-murdock-829d24.netlify.app/docs/guides/add-an-external-idp/openidconnect/main/#enterprise-identity-providers

### Resolves:
* [OKTA-488734](https://oktainc.atlassian.net/browse/OKTA-488734)

### CODE REVIEW

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md [hunk 1]
```diff
@@ -1,12 +1,11 @@
...
-
+* **Client ID**: Paste the app ID or client ID that you obtained when you configured the Identity Provider in the previous section.
+* **Authentication type**: Leave the default of **Client secret** or select **Public key/private key** to automatically generate a public and private key pair. The public key is available for download when you click **Finish**.
+* **Client Secret**: If you are using **Client secret** as the **Authentication type**, paste the secret that you obtained in the previous section.
 * **Authorize requests**: Select **Enable signed requests** to send request parameters to the OpenID provider as an encoded JWT instead of passing the parameters in the URL. <ApiLifecycle access="ea" />
-* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. <ApiLifecycle access="ea" />
+* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. If you are using **Public key/private key**, you must specify an encryption algorithm, for example: **RSA256**. <ApiLifecycle access="ea" />
```
Comment (venkatviswanathan-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md:8]: There is a whitespace between this and previous bullet

Comment (venkatviswanathan-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md:8]: And not encryption. It is Signing Algorithm

Comment (susanharper-okta) [reply packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md]: Unfortunately, it's the EA flag that causes the extra white space. I have made the other updates.

File: packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md [hunk 2]
```diff
@@ -1,12 +1,16 @@
...
+* **Authentication type**: Leave the default of **Client secret** or select **Public key/private key** to automatically generate a public and private key pair. The public key is available for download when you click **Finish**.
 
+    > **Note:** The **Public key/private key** option is an <ApiLifecycle access="ea" /> (Self-Service) feature. You can enable the feature for your org from the **Settings** > **Features** page in the Admin Console.
+
+* **Client Secret**: If you are using **Client secret** as the **Authentication type**, paste the secret that you obtained in the previous section.
 * **Authorize requests**: Select **Enable signed requests** to send request parameters to the OpenID provider as an encoded JWT instead of passing the parameters in the URL. <ApiLifecycle access="ea" />
-* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. <ApiLifecycle access="ea" />
+* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. If you are using **Public key/private key**, you must specify a signing algorithm, for example: **RSA256**. <ApiLifecycle access="ea" />
```
Comment (vanngo-okta) [inline packages/@okta/vuepress-site/docs/guides/add-an-external-idp/main/openidconnect/appidpinokta.md:11]:
```suggestion
* **Algorithm**: Select the algorithm to use for the signed requests from the dropdown list. If you are using the **Public key/private key** option, you must specify a signing algorithm, for example: **RSA256**. <ApiLifecycle access="ea" />
```

**Pull request merged**